src/utils/isTouchDevice.js

```diff
@@ -1,4 +1,4 @@
 export default function isTouchDevice() {
-  return !!(typeof window !== 'undefined' && 'ontouchstart' in window) ||
+  return !!(typeof window !== 'undefined' && window.ontouchstart != null) ||
```

Comment:
What value will `window.ontouchstart` have in a browser that supports touch?

However, https://hacks.mozilla.org/2013/04/detecting-touch-its-the-why-not-the-how/ seems to imply that we should be checking a few more things.